testbed/addons/tekton-dashboard/release.yaml

```diff
@@ -0,0 +1,492 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Can you add a comment or a script somewhere for how to programatically download or generate this? I don't have a problem checking it in if it's not available via helm somewhere.

Reply:
Spoke offline. Earlier it was programmatically installed through flux from tekton dashboard repo(https://github.com/awslabs/kubernetes-iteration-toolkit/pull/10/files#diff-12f8aaa63c2e28fe96b5d47b3651cfc21103ea346a1a91594da07aab395b5cb8L4) . As I mentioned in description, we are customizing this yaml for our testbed purposes to have a UI, hence checking in the yaml.

Reply:
I'll do CDK deploy locally without this change as workaround.